build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildBinPath)\Microsoft.Common.targets" />
+  <PropertyGroup>
+    <BuildDepends>
```

Comment:
So, is the expectation now that a dev should use `_Build` to do an actual build "in all cases"? And do we want them to use this target instead of `Build`?

If yes, then we should rename `_Build` -> `Build` and `Build` to something else more appropriate. That way if the user uses the (well known) `Build` target then they are always correct.

If no, then `ReallyBuild` sounds better than `_Build`. I can't think of many better names here. But `_Build` and `Build` just sound confusing to me.